src/Sentry.Unity/UnityLogException.cs

```diff
@@ -149,6 +164,215 @@ public SentryException ToSentryException()
             };
         }
 
+        private SentryException ConvertFullStackTrace()
+        {
+            if (LogStackTrace.StartsWith("0x")
+                && LogStackTrace.Contains("(Unity) StackWalker::GetCurrentCallstack"))
+            {
+                return ConvertWindowsFullStackTraceImpl();
+            }
+
+            return ConvertFullStackTraceImpl();
+        }
+
+        private static readonly Regex WindowsStackFramePattern = new Regex(

```

Comment:
Ideally we'd use the overload that takes a timeout (and passing the Compile flag is probably a good idea too):

```
public Regex (string pattern, System.Text.RegularExpressions.RegexOptions options, TimeSpan matchTimeout);
```
Timeout here for a second should be way more than enough, but a safenet for some bad input causing some really bad thing to happen